src/main/java/net/openhft/chronicle/core/util/ClassNotFoundRuntimeException.java

```diff
@@ -1,12 +1,7 @@
 package net.openhft.chronicle.core.util;
 
 public class ClassNotFoundRuntimeException extends RuntimeException {
-    public ClassNotFoundRuntimeException(ClassNotFoundException cause) {
+    public ClassNotFoundRuntimeException(Throwable cause) {
         super(cause);
     }
-
-    @Override
-    public synchronized ClassNotFoundException getCause() {
```

Comment:
ok to remove a public method without deprecating?

Reply:
So to deprecate it, I could just return null when the cause wasn't a ClassNotFoundException (that should be exceedingly rare). And we could remove it at the next release (to return Throwable again).
If I don't deprecate it, and remove it, it breaks compilation in Wire. But that's the only place it's used (in our libraries) and it's only been around since February so I can't imagine it's heavily depended on outside of our org.
What do you think @peter-k-lawrey should we play it safe or go cowboy on this. We can simultaneously release Wire and Core so the break shouldn't be evident.

Reply:
Scratch that I found a better way